--- pysrc-orig/pydevd.py	2013-04-16 17:45:28.592172220 -0400
+++ x86_64/eclipse/plugins/org.python.pydev.debug_1.6.4.2011010200/pysrc/pydevd.py	2013-04-16 17:46:33.613050456 -0400
@@ -1051,6 +1051,14 @@
             additionalInfo = PyDBAdditionalThreadInfo()
             t.additionalInfo = additionalInfo
   
+        # Hack for omniORB
+        try:
+            from omniORB import addWThreadHook
+        except ImportError:
+            pass
+        else:
+            addWThreadHook(self.dbg_trace)
+
         while not debugger.readyToRun: 
             time.sleep(0.1) # busy wait until we receive run command
 
